File: ios/PowpowNetworking.m
// PowpowNetworking.m

#import <React/RCTBridgeModule.h>

@interface RCT_EXTERN_MODULE(PowpowNetworking, NSObject)

RCT_EXTERN_METHOD(multiply:(float)a withB:(float)b
                 withResolver:(RCTPromiseResolveBlock)resolve
                 withRejecter:(RCTPromiseRejectBlock)reject)

@end

File: ios/PowpowNetworking.swift
//
//  PowpowNetworking.swift
//  PowpowNetworking
//
//  Created by lzp on 2022/3/12.
//  Copyright © 2022 Facebook. All rights reserved.
//

@objc(PowpowNetworking)
class PowpowNetworking: NSObject {

    @objc(multiply:withB:withResolver:withRejecter:)
    func multiply(a: Float, b: Float, resolve:RCTPromiseResolveBlock,reject:RCTPromiseRejectBlock) -> Void {
        resolve(a*b)
    }
}
